docs/fn-2.0/README.md

```diff
@@ -329,58 +329,70 @@ Picks runner to run it on and passes along event
 
 ## CLI changes
 
-### fn deploy
+### NEW IN 2.0
 
-func.yaml barely changes:
+The main changes here are related to [splitting functions and routes](https://github.com/fnproject/fn/issues/817).
+A route is now called a trigger and the route/trigger definition is no longer part of the function definition as it was before.
+
+### Function Definition File
+
+`func.yaml` barely changes, just removes route/trigger specific things. 
+
+A function does not know about triggers so it only has the information required to run the function.
 
 ```yaml
 function:
   name: yodawg
-  image: hub.docker.com/fnproject/hello
+
+  runtime: ruby
+  entrypoint: ruby func.rb
   version: 0.0.7
   memory: 42
+```
 
+`fn run` can run a function with this file only.
+
+### Trigger Definition File
+
+A `trigger.yaml` file defines the mapping from event sources to functions. A trigger
+binds one event source to one function, for instance an HTTP route to a function.
+
+```yaml
 trigger:
-  - name: myApp/sayhello
-    type: http
+  name: sayhello
+  type: http
+  # http specific param:
```

Comment:
let's just use name? the event source can strip the path to name format (i.e. strip `/`), redundant if anything and then we get into extensions for every event source type?

Reply:
i guess name has the same issue as the image if we do that. fuck me.

how about:

```
name: sayhello
source: /sayhello
```

`source` is generic and can be used across all event sources. e.g. a topic name, a schedule name, etc. where 'path' is domain specific to http.